ios/Capacitor/Capacitor/JSTypes.swift

```diff
@@ -9,6 +9,7 @@ extension Float: JSValue {}
 extension Double: JSValue {}
 extension NSNumber: JSValue {}
 extension Array: JSValue {}
+extension NSDate: JSValue {}
```

Comment:
This should be `Date`. Since these are Swift convenience wrappers, we should prefer the Swift data types wherever possible.
```suggestion
extension Date: JSValue {}
```